_includes/head.html

```diff
@@ -7,9 +7,20 @@
 ================================================== -->
   <meta charset="utf-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  {% unless site.env == "production" %}
-    <meta name="robots" content="noindex, nofollow">
-  {% endunless %}
+
+  {% if page.expiration_date %}
+    {% assign today_date = 'now' | date: '%s' %}
```

Comment:
Good question. `%s` converts the date to the number of "seconds since 1970". This allows us to compare the dates to see which is greater. 

Reference: The [documentation for Liquid's `date`](https://shopify.github.io/liquid/filters/date/) explains that they use the same formatting as [strftime](http://strftime.net/), which offers the `%s` conversion. 

I added a code comment in [bccb528](https://github.com/uswds/uswds-site/pull/2569/commits/bccb528ade3bc9a98dd6a562643b8ef86b51e625) for future reference.

Reply:
Thanks for the explanation and the comment; it'll be useful moving forward. 

I didn't see it in liquid docs and should've checked `strftime`.